modules/Configurator/views/view.edit.php

```diff
@@ -1,7 +1,4 @@
 <?php
```

Comment:
This file is not coveraged. (0%) It could be better, please add test for these parts where you having changes or additional code for this file at least.

Reply:
It seems the Configurator class has no tests whatsoever. I also cannot find any tests that specifically test functions in view.*.php files. 
Soooo... I'm creating a different pull request with tests for Configurator. It'll have at least a few methods tested so it's not completely useless. See pull #6701
I think I'll need some help testing some of the more complicated methods, as well has how to test the view.edit.php file.